requirements.txt

```diff
@@ -17,7 +17,7 @@ tensorboard==2.15.1
 pytorch-lightning==2.1.3
 
 # stable diffusion
--e git+https://github.com/kashif/diffusers.git@a3dc213#egg=diffusers # TODO: replace with official version
+diffusers==0.26.3
```

Comment:
I don't know why this fork was originally used, but I don't think it'd be needed anymore.

Reply:
This fork is needed for stable cascade support